.github/workflows/reusable-testing.yml

```diff
@@ -214,7 +219,7 @@ jobs:
         id: set-matrix
         run: |
           if [[ $FILE_EXISTS == 'true' ]]; then
-            echo "matrix=$(jq -c '.include |= map(with_entries(select(.key == "php"))) | .include |= map(select(.php >= "${{ inputs.minimum-php }}")) | .include |= unique_by(.php)' <<< $BASE_MATRIX)" >> $GITHUB_OUTPUT
+            echo "matrix=$(jq -c '.include |= map(select(.wp == "3.7").wp |= "${{ inputs.minimum-wp }}") | .include |= map(with_entries(select(.key == "php"))) | .include |= map(select(.php >= "${{ inputs.minimum-php }}")) | .include |= unique_by(.php)' <<< $BASE_MATRIX)" >> $GITHUB_OUTPUT
```

Comment:
This line here we don't need to change as we only return the `php` field for unit tests. Unit tests only run against specific PHP versions, as they don't run WP. So we can undo this.

Reply:
PR updated.